src/main/java/dk/sdu/se_f22/som2/IScoring.java

```diff
@@ -0,0 +1,34 @@
+package dk.sdu.se_f22.som2;
```

Comment:
This file should not be located here?!
Please insert all of your codes into sortingmodule/scoring